example/books/books.proto

```diff
@@ -3,12 +3,15 @@ syntax = "proto3";
 package books;
 
 import "github.com/kitt-technology/protoc-gen-graphql/graphql/graphql.proto";
-import "github.com/kitt-technology/protos-common/common.proto";
+import "github.com/kitt-technology/protoc-gen-graphql/example/common/common.proto";
 import "google/protobuf/wrappers.proto";
 import "google/protobuf/timestamp.proto";
 
 option go_package = "example/books;books";
 
+option (graphql.common_go_import) = "github.com/kitt-technology/protoc-gen-graphql/example/common";
+option (graphql.common_package_name) = "common";
```

Comment:
This might also be removable, it's the package name of the common proto so might be calculable?

Reply:
Maybe as we go through files we build a map of {package_name -> go_package}, then when we encounter a message we don't know how to deal with we can check that map and if it exists we know how to import it